modules/world/map/road_corridor.cpp

```diff
@@ -37,6 +38,22 @@ RoadCorridor::GetLeftRightLaneCorridor(const Point2d& pt) const {
     GetLaneCorridor(right_lane_id));
 }
 
+LaneCorridorPtr RoadCorridor::GetNearestLaneCorridor(const Point2d& pt) const {
```

Comment:
do we really need this? I assume this to be slower than GetCurrentLaneCorridor()

Reply:
In principal yes, but for example when working on the INTERACTION dataset, the reference points of the agents are sometimes slightly out of the lane corridor. `GetNearestLaneCorridor` first calls `GetCurrentLaneCorridor` and if a null pointer is returned, it searches for the nearest lane. So it should be only slower, in the exceptional case that an agent can not be assigned to a lane corridor.

Reply:
okay